src/Microsoft.AspNetCore.Http.Abstractions/QueryString.cs

```diff
@@ -121,12 +121,19 @@ public static QueryString Create(string name, string value)
             {
                 throw new ArgumentNullException(nameof(name));
             }
-            if (value == null)
+
+            return new QueryString($"?{UrlEncoder.Default.Encode(name)}={(string.IsNullOrEmpty(value) ? null : UrlEncoder.Default.Encode(value))}");
```

Comment:
nit: Add some indentation here, bit hard to read.

Reply:
I added a few spaces for you. I'm not sure exactly what you had in mind, so if you want further readability tweaks just let me know.